### DISCUSSION TITLE
List all unindexed files

### DISCUSSION THREAD

**Discussion by anthraxn8b** [photoprism/photoprism#4617] [Q&A]
How do I get a list of all NOT indexed files (because of errors)?

**Comment by omerdduran** [comment]
Hi . There is no command to print this directly, but this might help. To see the files that are not indexed (due to error) in PhotoPrism, you can search the Docker logs as follows:

```bash
docker compose logs | grep -i "error\|failed"
```

This command should show you indexing errors and failed files. I hope this helps.

**Reply by anthraxn8b** [reply]
THX. But is there sth. like an SQL query that I can use? Docker logs are not really my favorite 😁

**Reply by omerdduran** [reply]
Hello 👋

## Connecting to the Database

After connecting to the Mariadb container, you can connect to the database with this command

```sql
mariadb -u root -p your_database_name -- usually "photoprism"
```

**Important Note:** 
- This operation is recommended for advanced users only. Incorrect command usage can cause damage to the database.

## Viewing Error and Warning Messages
To view all error and warning messages in the database, you can use the following SQL query:

```sql
SELECT error_time, error_level, error_message 
FROM errors 
ORDER BY error_time DESC;
```

**Tip:** You can customize this query by adding WHERE conditions to view specific messages or filter by error level.

**Reply by anthraxn8b** [reply]
THX! I'll try my very best ;-)
